### PR TITLE
docs(publications): reorgs articles, ebook, and presos into Publicati…

### PR DESCRIPTION
**Pull request by dorbin**
…ons section

### CODE REVIEW

**Comment by dorbin** [comment]
@t0mpson

**Comment by dorbin** [comment]
#957 
#947

**Review by lwander — APPROVED** [review]

**Comment by tomaslin** [comment]
LGTM

**Review by tomaslin — APPROVED** [review]

File: _data/navigation.yml [hunk 1]
```diff
@@ -448,3 +441,11 @@ community:
     children:
       - title: FAQs
         url: /community/faqs/
+
+publications:
+  - title: eBook
```
Comment (demobox) [inline _data/navigation.yml:446]: [minor] Curious what the most common spelling here is, given https://en.wikipedia.org/wiki/E-book

**Review by demobox — APPROVED** [review]
Thanks, @dorbin!

**Comment by t0mpson** [comment]
Thanks everyone, it looks great now!

**Pull request merged**